.circleci/config.yml

```diff
@@ -156,6 +156,14 @@ jobs:
           root: *packages_workspace
           paths:
             - "*.tar.gz"
+  deploy_api_docs:
+    <<: *container_config
+    steps:
+      - checkout
+      - run:
+          command: |
+            make swagger-docs
```

Comment:
Why is this `make swagger-docs` needed - if only the swagger.json pushed?

Reply:
Yep @michalbagrowski we don't need this one.